android/quest/src/main/assets/configurations/view/register_view_configurations.json

```diff
@@ -35,7 +35,7 @@
     "showNewClientButton": true,
     "showSideMenu": false,
     "showBottomMenu": true,
-    "registrationForm": "3435",
+    "registrationForm": "9088",
```

Comment:
why do we need to set this, is this also for testing?

Reply:
Yes the configs are for testing.